packages/mobile/src/screens/AxiosExample.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { Component } from 'react';
+import { View, StyleSheet, ScrollView } from 'react-native';
+import News from '../components/News';
+import axios from 'axios';
+
+class AxiosExample extends Component{
+  constructor(props){
+		super(props);
+		this.state = {
+			news: []
+		}
+	}
+	componentDidMount() {
+    axios.get('https://newsapi.org/v2/everything?q=netflix&sortBy=publishedAt&apiKey=c04fd04b98d841d9ac3f0826f2497094')
+			.then(res => {
+				this.setState({ news: res.data.articles });
+			})
+  }
+  render(){
+    return(
+      <View style={styles.container}>
+        <ScrollView >
+          {
+            this.state.news.map((items, Id) =>
+              <News key={Id} data={items} />
```

Comment:
to na bank nie działa. brakuje return w map

Reply:
drugi parametr w map to nie id elementu a index w tablicy. jeśli item ma id to warto key dac jako id a nie jako index. index w key jest anty-wzorcem (!Zapamiętać pytaja o to czesto na rozmowach kwalifikacyjnych)